Elevate networks artifacts to top level and other

paulwizviz opened the pull request:
- Move and rename dev to addorg
- Move Kafka to top level and fix bug
- Move README and guides to docs

Signed-off-by: Paul Sitoh <paulwizviz@gmail.com>

paulwizviz committed:
Elevate networks artifacts to top level and other

- Move and rename dev to addorg
- Move Kafka to top level and fix bug
- Move README and guides to docs

Signed-off-by: Paul Sitoh <paulwizviz@gmail.com>

dmcarrington commented:
Looks fine so far. Are you intending to add further changes to this review as it's still in draft?

paulwizviz commented:
Yes. At least get the documentation up to scratch. Then I issue for a proper review.

paulwizviz committed:
Update documentation

- Remove content earmarked for the overview site
- Organised the docs section to make it appropriate for Github pages

Signed-off-by: Paul Sitoh <paulwizviz@gmail.com>

paulwizviz committed:
Update README

- Group network related documentation into a network folder
- Rewrite docs related to add organisation
- Add a landing page listing network examples
- Rewrite doc related to kafka network

Signed-off-by: Paul Sitoh <paulwizviz@gmail.com>

paulwizviz committed:
Bug fix addorg sub-project

Remnoved other non related artefacts.

Signed-off-by: Paul Sitoh <paulwizviz@gmail.com>

paulwizviz committed:
Reorganise folders

- Move crypto-configtx to top level folder
- Move fabric-ca to top level folder

Signed-off-by: Paul Sitoh <paulwizviz@gmail.com>

paulwizviz committed:
Update contribution

Signed-off-by: Paul Sitoh <paulwizviz@gmail.com>

dmcarrington reviewed (approved):
looks all good to me.
